src/main/java/io/pinecone/configs/PineconeConfig.java

```diff
@@ -202,20 +202,11 @@ public void validate() {
      * @return The user agent string.
      */
     public String getUserAgent() {
-        return buildUserAgent("pineconeClientVersion");
+        return buildUserAgent();
     }
 
-    /**
-     * Builds the user agent string for the Pinecone client's gRPC requests.
-     *
-     * @return The user agent string for gRPC requests.
-     */
-    public String getUserAgentGrpc() {
```

Comment:
removed this method since netty-java-grpc/ is already appended to the user-agent string